css/css-ui-3/text-overflow-014.html

```diff
@@ -0,0 +1,47 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: styling the ellipsis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#ellipsing-details">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="The ellipsis is styled according to the block. ">
+<style>
+.test {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 2em;
+  margin-left: -1em;
+  white-space:pre;
+  line-height: 1;
+  font-size: 100px;
+  font-family: ahem;
+  color: green;
+
+  /* 2em width and starting with a space
+     because the handling of the ellipsis of the first character on the line is special,
+     and therefore we need to have the thing we want to test in non first position.
+     Shifting 1em left to align with the usual green square.
+   */
+  width: 2em;
+  margin-left: -1em;
+}
+.red {
+  position: absolute;
+  z-index: -1;
+  width: 100px;
+  height: 100px;
+  background: red;
+}
+span {
+	font-family: serif;
+	color: red;
+	line-height: 0; /* not clear that this is needed based on specs,
+			   but without it, line height calculations go
+			   wrong (too tall) in some browsers. */
+}
+</style>
+
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+<div class=red></div>
+<div class=test> <span>                    </span></div>
```

Comment:
The elided text here should be x's not spaces, so that they will be visibly red if not elided.

I also think this test would be clearer if written based on the 50px template, since then we just put a little extra text in it instead of fussing with negative sizes, but ymmv.

I do think you can turn the abspos into a red background on the test div, though, right?